openforcefield/tests/test_toolkits.py

```diff
@@ -2817,6 +2817,55 @@ def test_check_n_conformers(self):
 class TestToolkitRegistry:
     """Test the ToolkitRegistry class"""
 
+    def test_register_empty_toolkit(self):
```

Comment:
This test could potentially be dropped if we want `ToolkitRegistry()` to not be allowed, i.e. cannot construct an empty one. Personally I think the flexibility of being able to start with an empty registry is worthwhile and the default registration of toolkits should be handled with whatever update to `GLOBAL_TOOLKIT_*` we figure out.